include/modules/alsa.hpp

```diff
@@ -67,6 +67,7 @@ namespace modules {
     map<control, control_t> m_ctrl;
     int m_headphoneid{0};
     bool m_mapped{false};
+    bool m_unmuteOnScroll{false};
```

Comment:
```suggestion
    bool m_unmute_on_scroll{false};
```

We're using snake_case for variable names, please also update in other module